sw/device/tests/power_virus_systemtest.c

```diff
@@ -1222,6 +1221,11 @@ static void max_power_task(void *task_parameters) {
   mmio_region_write32(gpio.base_addr, GPIO_MASKED_OUT_LOWER_REG_OFFSET,
                       gpio_on_reg_val);
 
+  mmio_region_write32(hmac.base_addr, HMAC_CMD_REG_OFFSET,
```

Comment:
@timothytrippel 
This change was made for closed source GLS and it passes open source test as well.
Do you think any potential issue with this?

Reply:
if it passes in open-source too then we should be ok.